antlr-kotlin-examples-mpp/src/commonAntlr/antlr

```diff
@@ -0,0 +1 @@
+../../../antlr-kotlin-examples-jvm/src/main/antlr
```

Comment:
I do not understand this file and the two following

Reply:
Hi @tfomassetti,
they are just a symlinks